distribution/lib/Standard/Google_Api/0.0.0-dev/src/Google_Analytics.enso

```diff
@@ -0,0 +1,84 @@
+from Standard.Base import all
+import Standard.Base.Data.Array_Proxy.Array_Proxy
+import Standard.Base.Metadata.Display
+from Standard.Base.Metadata import make_single_choice, Widget
+from Standard.Base.Metadata.Widget import Single_Choice, Vector_Editor
+from Standard.Table import Table
+import project.Google_Credential.Google_Credential
+from project.Internal.Google_Credential_Internal import all
+
+polyglot java import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+polyglot java import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+polyglot java import com.google.api.client.json.gson.GsonFactory
+polyglot java import com.google.api.services.sheets.v4.Sheets
+polyglot java import com.google.api.services.sheets.v4.SheetsScopes
+polyglot java import java.util.Collections
+polyglot java import com.google.analytics.data.v1beta.BetaAnalyticsDataClient
+polyglot java import com.google.analytics.data.v1beta.BetaAnalyticsDataSettings
+polyglot java import com.google.analytics.data.v1beta.DateRange
+polyglot java import com.google.analytics.data.v1beta.Dimension
+polyglot java import com.google.analytics.data.v1beta.Metric
+polyglot java import com.google.analytics.data.v1beta.Row
+polyglot java import com.google.analytics.data.v1beta.RunReportRequest
+polyglot java import com.google.analytics.data.v1beta.RunReportResponse
+polyglot java import com.google.auth.oauth2.GoogleCredentials
+polyglot java import com.google.api.gax.core.FixedCredentialsProvider
+polyglot java import com.google.api.gax.core.CredentialsProvider
+
+
+## PRIVATE
```

Comment:
Could these `## PRIVATE` methods be moved to `Internal` and `private` module?

Reply:
I'm not certain, but I suspect the answer is no as they are used by the GUI to generate the drop down widgets.